server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java

```diff
@@ -162,11 +163,12 @@ public static TabletFiles updateTabletVolumes(ServerContext context, ServiceLock
     }
 
     for (Entry<StoredTabletFile,DataFileValue> entry : tabletFiles.datafiles.entrySet()) {
-      String metaPath = entry.getKey().getMetaUpdateDelete();
```

Comment:
I figured when we start to use the ranges we will need to add tests for things like this to verify they are properly used. It would obviously be pretty bad if a range got dropped.

Reply:
Yeah looking at this I was thinking what is the minimal amount of test and code changes that is the most maintainable and ensures we have this correct.  It seems like we may need a lot of tests that verify lots of operations work properly when there is a range present, I am not sure if there is anyway to avoid that.  We may want to open follow on issues for each test that is needed.